preprocessing-scripts/copy_images_to_correct_location.py

```diff
@@ -90,7 +90,7 @@ def main(arguments: argparse.Namespace) -> None:
     # missing_ids = pd.read_csv(
     #     missing_id_path / "Updated_Missing_IDs.csv", index_col=0
     # ).index.to_list()
-    # data = data.loc[missing_ids]
+    # preprocessing = preprocessing.loc[missing_ids]
```

Comment:
```suggestion
    # data = data.loc[missing_ids]
```
The missing IDs are not really relevant anymore, right?

Reply:
Yep - I have removed them